### PR TITLE
_reportSanityChecks: more comments and straightforward calculations

### PR DESCRIPTION
**Pull request by dechjo**
Updated `_reportSanityChecks` code for more readability.

### CODE REVIEW

**Review by skozin — APPROVED** [review]

File: contracts/0.4.24/oracle/LidoOracle.sol [hunk 1]
```diff
@@ -621,18 +611,27 @@ contract LidoOracle is ILidoOracle, AragonApp {
...
-            uint256 allowedBeaconBalanceDecrease =
-                ALLOWED_BEACON_BALANCE_RELATIVE_DECREASE_POSITION.getStorageUint256().mul(_preTotalPooledEther);
-            uint256 loss = uint256(10000).mul(delta);
-            require(loss <= allowedBeaconBalanceDecrease, "ALLOWED_BEACON_BALANCE_DECREASE");
+        if (_postTotalPooledEther >= _preTotalPooledEther) {
+            // increase                        = _postTotalPooledEther - _preTotalPooledEther,
+            // relativeIncrease                = increase / _preTotalPooledEther,
+            // annualRelativeIncrease          = relativeIncrease / (timeElapsed / 365 days),
+            // annualRelativeIncreasePermyriad = annualRelativeIncrease * 10000
+            uint256 allowedAnnualRelativeIncreasPermyriad =
```
Comment (ujenjt) [inline contracts/0.4.24/oracle/LidoOracle.sol:619]: We are using BP (basis points) in another contracts (e.g Lido.sol, StETH.sol)
Permyriad is very rare word and needs to be googled any time

**Review by ujenjt — APPROVED** [review]
LGTM

File: contracts/0.4.24/oracle/LidoOracle.sol [hunk 2]
```diff
@@ -621,18 +611,27 @@ contract LidoOracle is ILidoOracle, AragonApp {
...
+                ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_INCREASE_POSITION.getStorageUint256();
+            // check that annualRelativeIncreasePermyriad <= allowedAnnualRelativeIncreasePermyriad
+            require(uint256(10000 * 365 days).mul(_postTotalPooledEther - _preTotalPooledEther) <=
+                    allowedAnnualRelativeIncreasPermyriad.mul(_preTotalPooledEther).mul(_timeElapsed),
```
Comment (skozin) [inline contracts/0.4.24/oracle/LidoOracle.sol:623]: There's a typo here: allowedAnnualRelative**Increas**Permyriad, should be Increase. I'd name it `allowedAnnualRelativeIncreaseBP`, too—at least for consistency with other contracts.

**Review by skozin — CHANGES_REQUESTED** [review]

File: contracts/0.4.24/oracle/LidoOracle.sol (hunk 1 above)
Comment (dechjo) [reply contracts/0.4.24/oracle/LidoOracle.sol]: Replaced Permyriad -> Bp (Basis points), thanks

File: contracts/0.4.24/oracle/LidoOracle.sol (hunk 2 above)
Comment (dechjo) [reply contracts/0.4.24/oracle/LidoOracle.sol]: agree, thanks!

**Review by skozin — APPROVED** [review]

**Pull request merged**